docs/usage-as-end-user.md

```diff
@@ -96,9 +96,14 @@ serverGroupConfig:
   policy: soft-anti-affinity
 ```
 
-Currently, the only configuration available is the `serverGroupConfig`. If this option is set, then a new server group will be created with the configured policy and all machines managed by this worker pool will be assigned as members of the group.
-
-Please note that the available options for `serverGroupConfig` are defined in the provider specific section of `CloudProfile`.
+When you specify `serverGroupConfig` in your worker configuration, a new server group will be created with the configured policy and all machines managed by this worker will be assigned as members of the created server group.
```

Comment:
As an end-user I would be interested if that happens in-place of whether my existing machines will be terminated/replaced by new machines (i.e., rolling update)?